classes/Task/Upgrade/BackupDb.php

```diff
@@ -39,6 +41,8 @@ class BackupDb extends AbstractTask
 {
     const TASK_TYPE = 'upgrade';
 
+    const MAX_SIZE_PER_INSERT_STMT = 950000;
+
     /**
      * @throws Exception
      */
```

Comment:
The run function still remains very hard to read... 219 lines long for a function is too much, even after extracting several private functions. 

It's a good start :)

Reply:
Indeed. Unfortunately because of the local variable that can be modified inside the loops (such as the time spent, the quantity written in file...), I wasn't able to split everything without redoing the whole class.